controllers/users.js

```diff
@@ -223,7 +249,9 @@ class Users {
     }
 
     // create a JWT token
-    const token = await jwt.sign({ email }, process.env.SECRET, { expiresIn: '2h' });
+    const token = await jwt.sign({ email }, process.env.SECRET, {
+      expiresIn: '2h'
+    });
     // send email using SendGrid
     sgMail.setApiKey(process.env.SENDGRID_API_KEY);
```

Comment:
Just check if you have the actual files because this option of sending email is no longer in controller/users.js

Reply:
I can see that you have conflicts on that file controllers/users.js fix them

Reply:
Thank you I had to rebase with latest changes